src/subdataframe/subdataframe.jl

```diff
@@ -5,8 +5,7 @@
 ##
 ##############################################################################
 
-if VERSION >= v"0.6.0-dev.2643"
-    include_string("""
+include_string("""
```

Comment:
This was is funny, even though it probably doesn't qualify as `bad bot`.

Reply:
Yeah, it'd be difficult to programmatically determine whether an `include_string` is no longer necessary. We can always push a commit to this branch to get rid of it.